pkgs/development/python-modules/manticore/default.nix

```
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, capstone
+, crytic-compile
+, fetchFromGitHub
+, pkgs
+, ply
+, prettytable
+, pyelftools
+, pyevmasm
+, pysha3
+, pytestCheckHook
+, pyyaml
+, rlp
+, setuptools
+, stdenv
+, unicorn
+, wasm
+, yices
+}:
+
+buildPythonPackage rec {
+  pname = "manticore";
+  version = "0.3.4";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "manticore";
+    rev = "4712491d379f1fa6c8faa2d38809877e31259f78";
+    sha256 = "0i9653013wvnj6y7f7i7vyj97250wd1qspr9ylpb1pnhi3c23gi0";
+  };
+
+  propagatedBuildInputs = [
+    crytic-compile
+    ply
+    prettytable
+    pyevmasm
+    pysha3
+    pyyaml
+    rlp
+    setuptools
+    wasm
+  ] ++ lib.optionals (stdenv.isLinux) [
+      capstone
+      pyelftools
+      unicorn
+    ];
```

Formatting: extra spaces/tabs
```suggestion
  ] ++ lib.optionals (stdenv.isLinux) [
    capstone
    pyelftools
    unicorn
  ];
```